src/lib/structures/modelMap.ml

```diff
@@ -80,24 +80,49 @@ module V = Set.Make
 
 type key = P.key
 type elt = V.t
-type t = V.t P.t
 
-let add p v mp =
-  let prof_p = try P.find p mp with Not_found -> V.empty in
-  if V.mem v prof_p then mp
-  else P.add p (V.add v prof_p) mp
+type t = {
+  values : V.t P.t;
+  suspicious : bool;
+}
 
-let iter f =
+let is_suspicious_name hs =
+  match Hstring.view hs with
+  | "@/" | "@%" | "@*" -> true
+  | _ -> false
+
+(* The model generation is not ready for FPA and Bitvector theories. *)
```

Comment:
s/not ready/known to be incomplete

Reply:
Please don't resolve conversations asking for a change without making the corresponding change.

Reply:
Sorry, I thought I did it.